jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -121,6 +121,7 @@ def process(self, input_data):
                 'resample_data': self.resample_data,
                 'good_bits': self.good_bits,
                 'make_output_path': self.make_output_path,
+                'mk_output_list': True
```

Comment:
Is it expected that we want this controlled via the `stpipe` infrastructure (eg set by the user via a command line option)? If not, would an attribute work?

Reply:
I do not envision giving control of this via stpipe. I made up this parameter at this point to pass it to the resample step and get the output filenames of the files created, but there are some cases in which we don't want this, so the parameter is turned off. Either way, this should be done under the hood, the user has control over the keeping of the output files.